index.bs

```diff
@@ -746,29 +747,28 @@ authorizing an authenticator.
 </div>
 
 
-### Use an existing credential to make an assertion - PublicKeyCredential's `[[DiscoverFromExternalSource]](options)` method ### {#getAssertion}
+### Use an existing credential to make an assertion ### {#getAssertion}
 
-<div link-for-hint="PublicKeyCredential/[[DiscoverFromExternalSource]](options)">
-The <dfn for="PublicKeyCredential" method>\[[DiscoverFromExternalSource]](options)</dfn> method is used to discover and use an
-existing [=public key credential=], with the user's consent. The script optionally specifies some criteria to indicate what
-credentials are acceptable to it. The user agent and/or platform locates credentials matching the specified criteria, and guides
-the user to pick one that the script will be allowed to use. The user may choose not to provide a credential even if one is
-present, for example to maintain privacy.
+[RPS] call <code><a idl for="CredentialsContainer" lt="get()">navigator.credentials.get({publicKey:..., ...})</a></code> to
+discover and use an existing [=public key credential=], with the user's consent. The script optionally specifies some criteria
+to indicate what [=credential sources=] are acceptable to it. The user agent and/or platform locates [=credential sources=]
+matching the specified criteria, and guides the user to pick one that the script will be allowed to use. It then uses
+[[#op-get-assertion]] to sign a [RP]-provided challenge and other collected data into an assertion, which is used as a
+[=credential=]. The user may choose not to provide a credential even if one is present, for example to maintain privacy.
 
-Note: This algorithm is synchronous; the {{Promise}} resolution/rejection is handled by
-{{CredentialsContainer/get()|navigator.credentials.get()}}.
-
-This method accepts a single argument:
+The {{CredentialsContainer/get()}} implementation calls
+<code>PublicKeyCredential.{{PublicKeyCredential/[[CollectFromCredentialStore]]()}}</code> to collect any [=credentials=] that
+should be available without [=user mediation=] (roughly this specification's [=test of user presence=]), and if it doesn't find
```

Comment:
s/roughly/roughly:/  or  /roughly,/

I would also mention user verif here:
...specification's [=test of user presence=] or [=user verification=]...

..but rather than that, I suggest just using [=authorization gesture=] instead of the more specific types of [=authorization gesture=].

Reply:
Sure.